src/lib/checkbox/checkbox.ts

```diff
@@ -303,6 +303,10 @@ export class MatCheckbox extends _MatCheckboxMixinBase implements ControlValueAc
     this._changeDetectorRef.markForCheck();
   }
 
+  getAriaChecked() {
+    return this.checked ? true : (this.indeterminate ? 'mixed' : false);
```

Comment:
Return type should be `'true' | 'false' | 'mixed'` (since the attr should always be a string)

Reply:
Done. Thanks for review!